.github/workflows/ci.yml

```diff
@@ -127,10 +127,10 @@ jobs:
 
       - name: Run sanity tests
         run: make test_collection_sanity
-        env:
+        # env:
           # needed due to cgroupsv2. This is fixed, but a stable release
           # with the fix has not been made yet.
-          ANSIBLE_TEST_PREFER_PODMAN: 1
+          # ANSIBLE_TEST_PREFER_PODMAN: 1 Commenting due to issue with crun
```

Comment:
I would just nix the lines entirely. We don't _need_ to use podman here, it was done as a workaround when docker was broken originally, but now that's fixed.